js/test/components/operatorSignIn/operatorSignInModal.test.tsx

```diff
@@ -72,6 +72,7 @@ describe("OperatorSignInModal", () => {
     await userEvent.click(view.getByRole("button", { name: "OK" }));
 
     expect(view.getByText("Step 2 of 2")).toBeInTheDocument();
+    expect(view.getByRole("textbox")).toHaveValue(""); // Not pre-filled if manually typed
```

Comment:
**question:** Hmm, this isn't a change that you made, but now I'm confused: why is it going through the type workflow here? At the top of the file we mock `useNfcSupported` to return `true` and it's only overridden in one other test, with a `mockReturnValueOnce` for that matter.

Reply:
Ah! The two workflows are not mutually exclusive. Even when `useNfc` is returning `reading`, you can type a badge number and proceed that way.

Reply:
Ohh, right, of course! Sorry for the confusion.

Reply:
No prob 😃